src/components/Planner/PlannerEvents/PlannerEvents.jsx

```diff
@@ -1,157 +1,82 @@
+import { useState, useEffect, useContext } from 'react';
 import s from '../PlannerLayout.module.css';
-import {
-  TableContainer,
-  Table,
-  Thead,
-  Tr,
-  Th,
-  Tbody,
-  Text,
-  Button,
-  Input,
-  InputGroup,
-  InputLeftElement,
-  Stack,
-  Select,
-} from '@chakra-ui/react';
-import { Search2Icon, AddIcon } from '@chakra-ui/icons';
-import { useState } from 'react';
+import { Text, Button, Heading, Box } from '@chakra-ui/react';
+import { AddIcon } from '@chakra-ui/icons';
 import AddEventOverlay from './AddEventOverlay';
+import Catalog from '../../../pages/Catalog/Catalog';
+// import PropTypes from 'prop-types';
+import { DayIdContext } from '../../../pages/PublishedSchedule/AddDayContext';
+import { NPOBackend } from '../../../utils/auth_utils';
 
 const PlannerEvents = () => {
   const [overlayIsVisible, setOverlayIsVisible] = useState(false);
+  const [isVisible, setIsVisible] = useState(true);
+  const [existingEventData, setExistingEventData] = useState({});
+  const [dateHeader, setDateHeader] = useState('');
+  const { dayId } = useContext(DayIdContext);
+
+  useEffect(() => {
+    const getDayData = async () => {
+      try {
+        const response = await NPOBackend.get(`/day/${dayId}`);
+        // const [locationPart] = response.data[0].location.split('T');
+        // setLocationPart(locationPart);
+        const [datePart] = response.data[0].eventDate.split('T');
+        setDateHeader(datePart);
+      } catch (error) {
+        console.error(error);
+      }
+    };
+    getDayData();
+    console.log(dateHeader);
+  }, [dayId]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'dateHeader'. Either include it or remove the dependency array.